CoroutineTemplate/app/src/main/java/co/nimblehq/coroutine/ui/screens/second/SecondFragment.kt

```diff
@@ -1,24 +1,30 @@
 package co.nimblehq.coroutine.ui.screens.second
 
-import androidx.fragment.app.viewModels
+import android.os.Bundle
+import android.view.*
+import androidx.annotation.CallSuper
+import androidx.fragment.app.Fragment
 import androidx.navigation.fragment.navArgs
 import co.nimblehq.coroutine.R
-import co.nimblehq.coroutine.ui.base.BaseFragment
 import dagger.hilt.android.AndroidEntryPoint
 import kotlinx.android.synthetic.main.fragment_second.*
 
 @AndroidEntryPoint
-class SecondFragment : BaseFragment() {
-
-    override val layoutRes: Int = R.layout.fragment_second
-
-    private val viewModel by viewModels<SecondViewModel>()
+class SecondFragment : Fragment() {
 
     private val args: SecondFragmentArgs by navArgs()
 
-    override fun setupView() {
-        tvMessage.text = args.bundle.message
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        return inflater.inflate(R.layout.fragment_second, container, false)
     }
 
-    override fun bindViewModel() {}
+    @CallSuper
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        tvMessage.text = args.bundle.message
+    }
```

Comment:
not sure what is the reason why do we skip using base fragment? Using base fragment would help us much to handle most of the common steps in the base level 💭

Reply:
Totally agree @luongvo we have  a plan to include base fragment in another PR and all of dependencies e.g BaseFragmentCallback that why we skip this part for now 🙏 What do you think? @Tuubz

Reply:
We could keep it for a separate ticket as well, but I think having the changes for base-fragment & base-fragment callbacks would be pretty straight-forward and easy to understand. So, I'm fine with bringing back those classes in this PR 🙏 

(I think I should have done it in the previous PR: #18 🙈 )